File: Codigo/App_mobile/techapps_admin/lib/config/entity/visita.dart

import 'package:techapps_admin/config/entity/residente.dart';


class Visita {
  final int id;
  final String rut;
  final String nombre;
  final String apellido;
  final Residente residente;

  Visita({
    required this.id,
    required this.rut,
    required this.nombre,
    required this.apellido,
    required this.residente,
  });
}

File: Codigo/Springboot/msmultas/src/main/java/com/duoc/msmultas/model/entity/Visita.java
package com.duoc.msmultas.model.entity;

import jakarta.persistence.*;
import lombok.*;

import java.io.Serializable;

@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString
@Builder
@Entity
@Table(name = "visita")
public class Visita implements Serializable {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(name = "rut", nullable = false, unique = true, length = 9)
    private String rut;

    @Column(name = "nombre", nullable = false, length = 25)
    private String nombre;

    @Column(name = "apellido", nullable = false, length = 25)
    private String apellido;

}
